qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -57,3 +61,50 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown sample template action: %s' % action)
         self.write(result)
+
+
+def build_sample_summary(study_id, user_id, visible=None):
```

Comment:
The prep template is a bit of an issue right now, since multiple prep templates can have the same column name and not all samples will be on each column. Because of this issue, I'm only showing sample template columns for now.